gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy

```diff
@@ -21,16 +21,35 @@ import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
 import com.palantir.gradle.dist.service.JavaServiceDistributionPlugin
 import groovy.transform.EqualsAndHashCode
 import groovy.transform.ToString
+import java.nio.file.Files
 import org.gradle.api.DefaultTask
 import org.gradle.api.file.FileCollection
-import org.gradle.api.tasks.*
-
-import java.nio.file.Files
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
 
 class LaunchConfigTask extends DefaultTask {
 
     static final List<String> tmpdirJvmOpts = ['-Djava.io.tmpdir=var/data/tmp']
 
+    static final Map<String, List<String>> gcProfiles = [
+            'throughput'   : ['-XX:+UseParallelOldGC'],
+            'response-time': ['-XX:+UseParNewGC',
+                              '-XX:+UseConcMarkSweepGC',
+                              '-XX:+CMSParallelInitialMarkEnabled',
+                              '-XX:+CMSParallelRemarkEnabled',
+                              '-XX:+UseCMSInitiatingOccupancyOnly',
+                              '-XX:CMSInitiatingOccupancyFraction=68',
```

Comment:
would love a way to make this percentage configurable as different workloads may need adjusted

Reply:
so this still can be overridden by anything in launcher-custom. Not sure what's the best api here. I have never seen it set to anything else internally